### PR TITLE
Bl-11583 draft publishing

### PR DESCRIPTION
**Pull request by nabalone**
This change is [Reviewable](https://reviewable.io/reviews/BloomBooks/BloomDesktop/5864)

### CODE REVIEW

File: src/BloomBrowserUI/publish/PDFPrintPublish/PDFPrintFeaturesGroup.tsx [hunk 1]
```diff
@@ -151,7 +151,7 @@ export const PDFPrintFeaturesGroup: React.FunctionComponent<{
                             }}
                         >
                             <ApiCheckbox
-                                english="Full Bleed"
+                                label="Full Bleed"
```
Comment (nabalone) [inline src/BloomBrowserUI/publish/PDFPrintPublish/PDFPrintFeaturesGroup.tsx:154]: I manually tested some but not all of the ApiCheckboxes that got touched. This one, for example, I wasn't sure how to enable

File: src/BloomBrowserUI/publish/commonPublish/PublishVisibilityGroup.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import { useL10n } from "../../react_components/l10nHooks";
+import { Span } from "../../react_components/l10nComponents";
+import { ApiCheckbox } from "../../react_components/ApiCheckbox";
+
+export const PublishVisibilityGroup: React.FunctionComponent<{
+    onChange?: () => void;
+}> = props => {
+    return (
+        <SettingsGroup
+            label={useL10n("Visibility", "PublishTab.Upload.Visibility")}
```
Comment (nabalone) [inline src/BloomBrowserUI/publish/commonPublish/PublishVisibilityGroup.tsx:14]: Is there anything else I need to do to make localization work for new words?

**Review by andrew-polk — COMMENTED** [review]
Great job! A few things to look at.

> Reviewed 9 of 9 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/BloomBooks/BloomDesktop/5864#-:-NVp4qeA2sECBPQhKtrv:b-b4wcz5)* status: all files reviewed, 2 unresolved discussions (waiting on @nabalone)

___
*[`src/BloomBrowserUI/publish/commonPublish/PublishVisibilityGroup.tsx` line 14 at r1](https://reviewable.io/reviews/BloomBooks/BloomDesktop/5864#gh-1199184022:-NVp44jY-icf5wmyYDFa:b-tfcy42) ([raw file](https://github.com/BloomBooks/BloomDesktop/blob/23e543bfdf0d0567b2e89aac0cf27a12b4e4be6c/src/BloomBrowserUI/publish/commonPublish/PublishVisibilityGroup.tsx#L14)):*
<details><summary><i>Previously, nabalone (Noel) wrote…</i></summary><blockquote>

Is there anything else I need to do to make localization work for new words?
</blockquote></details>

Yes, the two new strings need to be added to an .xlf file. In this case, `\DistFiles\localization\en\BloomMediumPriority.xlf`.
See other ones added for 5.5. You'll want a similar comment (for 5.6) and `translate='no'` meaning we are not yet ready for translators to translate them.
We try to keep the IDs alphabetized.
___
*[`src/BloomBrowserUI/publish/PDFPrintPublish/PDFPrintFeaturesGroup.tsx` line 154 at r1](https://reviewable.io/reviews/BloomBooks/BloomDesktop/5864#gh-1199183683:-NVp3EoA5Jo8kDbdRW_L:b-aexkvv) ([raw file](https://github.com/BloomBooks/BloomDesktop/blob/23e543bfdf0d0567b2e89aac0cf27a12b4e4be6c/src/BloomBrowserUI/publish/PDFPrintPublish/PDFPrintFeaturesGroup.tsx#L154)):*
<details><summary><i>Previously, nabalone (Noel) wrote…</i></summary><blockquote>

I manually tested some but not all of the ApiCheckboxes that got touched. This one, for example, I do not know how to enable
</blockquote></details>

That's fine. I don't think you need to test them all. Prop name change is pretty innocuous.
And since you're just passing it through, the prop type change is pretty innocuous, too.
___
*[`src/BloomExe/web/controllers/PublishApi.cs` line 272 at r2](https://reviewable.io/reviews/BloomBooks/BloomDesktop/5864#-NVp3dBLBtzn38Y1reaq:-NVp3dBLBtzn38Y1rear:b3rloy7) ([raw file](https://github.com/BloomBooks/BloomDesktop/blob/7eaa439b6c17571c6fa5c42071649bc7d4d16993/src/BloomExe/web/controllers/PublishApi.cs#L272)):*
> ```Smalltalk
> 				(writeRequest, value) =>
> 				{
> 					System.Console.Write("writerequest");
> ```

These System.Console lines add a slight overhead to the execution and also clutter the code a little.
I suggest we remove them.

**Comment by andrew-polk** [comment]
*[`src/BloomBrowserUI/publish/commonPublish/PublishVisibilityGroup.tsx` line 10 at r2](https://reviewable.io/reviews/BloomBooks/BloomDesktop/5864#-NVp5ogC4fgMeuVC70vK:-NVp5ogC4fgMeuVC70vL:b-xghe10) ([raw file](https://github.com/BloomBooks/BloomDesktop/blob/7eaa439b6c17571c6fa5c42071649bc7d4d16993/src/BloomBrowserUI/publish/commonPublish/PublishVisibilityGroup.tsx#L10)):*
> ```XML
> 
> export const PublishVisibilityGroup: React.FunctionComponent<{
>     onChange?: () => void;
> ```

Looks like we don't need this onChange prop.

**Review by andrew-polk — COMMENTED** [review]
> Reviewed 3 of 3 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/BloomBooks/BloomDesktop/5864)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @nabalone)

**Review by andrew-polk — COMMENTED** [review]
> Dismissed @nabalone from a discussion.
> *[Reviewable](https://reviewable.io/reviews/BloomBooks/BloomDesktop/5864)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @nabalone)

**Pull request merged**